src/Page/Submission/Team1/Team1.js

```diff
@@ -2,18 +2,604 @@
+                    threshold: null
+                  }
+                },
+                series: [{
+                                  type: 'area',
+                                  name: top5Arr[i][0],
+                                  data: this.matchDateAndValues(this.getdates(top5Arr[i][0]), this.getcloses(top5Arr[i][0]),0)
+                              }, {
+                                  name: 'Simple Moving Average, 19-day period',
+                                  data: this.matchDateAndValues(this.getdates(top5Arr[i][0]), this.sma19(top5Arr[i][0]),19)
+                              },{
+                                name: 'Exponential Moving Average, 19-day period',
+                                data: this.matchDateAndValues(this.getdates(top5Arr[i][0]), this.ema(top5Arr[i][0], 19),19)
+                              },{
+                                name: 'Exponential Moving Average, 39-day period',
+                                data: this.matchDateAndValues(this.getdates(top5Arr[i][0]), this.ema(top5Arr[i][0], 39),39)
+                              }]
+               
+        }
+        optionsLineArr.push(tempOptions);
+      }
+      return optionsLineArr;
+    }
+   
+    getPortfolioDistributionArr(top5Arr) {
+      let boost = 0;
+      let distArr = [];
+      if (top5Arr[0][1]<1){
+        boost = 1-top5Arr[0][1];
+      }
+      let sum = 0;
+      for (let i=0;i<5;i++) {
+        sum += Math.sqrt(top5Arr[i][1]);
+      }
+      for (let i=0;i<5;i++){
+        distArr.push(Math.sqrt(top5Arr[i][1]) / sum);
+      }
+      return distArr;
+    }
+ 
+ roundoff(Port) {
+    let temp = Port;
+    for (let i=0; i<Port.length; i++)
+    {
+      temp[i] = (Math.round(temp[i] * 100));
+    }
+    return temp;
+ }
+ 
   render() {
+   
+    let top5Arr = this.top5();
+    var optionsLineArr = this.makeOptionsLineArr();
+    for (let i=0; i<5; i++){
+      console.log(top5Arr[i]);
+    }
+    let portfolioDistributionArr = this.getPortfolioDistributionArr(top5Arr);
+    let rounded = this.roundoff(portfolioDistributionArr);
+    optionsPie['series'] = [{
+      name: 'Stock',
+          colorByPoint: true,
+          data: [{
+            name: top5Arr[4][0],
+            y: portfolioDistributionArr[4]
+          }, {
+            name: top5Arr[3][0],
+            y: portfolioDistributionArr[3]
+          }, {
+            name: top5Arr[2][0],
+            y: portfolioDistributionArr[2]
+          }, {
+            name: top5Arr[1][0],
+            y: portfolioDistributionArr[1]
+          }, {
+            name: top5Arr[0][0],
+            y: portfolioDistributionArr[0]
+          }]
+    }];
+   
     return (
-      <div className='container'>Team 1</div>
-    )
-  }
-}
+    <div className='container'>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"/>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"/>
```

Comment:
a library Reactstrap was imported for you at the beginning. I will suggest to look into that if possible. It is basically bootstrap in React. So you don't have to import all these stuff.